modules/appeals_api/app/workers/appeals_api/daily_error_report.rb

```diff
@@ -8,8 +8,12 @@ class DailyErrorReport
     # Only retry for ~8 hours since the job is run daily
     sidekiq_options retry: 11, unique_for: 8.hours
 
+    RECIPIENTS_FILENAME = 'decision_review_error_report_daily.yml'
```

Comment:
What about letting the ReportRecipientsReader respond with the parsed recipients here? It could store the mailing list path for itself. Maybe something like:

```ruby
RECIPIENTS = ReportRecipientsReader.load_recipients(:error_report_daily).freeze
```

Also, I don't think we need to prefix our mailing list filenames with `decision_review` - since it's in our own module's config file we know it's a part of AppealsApi/Decision Review. We prefix our Flipper flags with `decision_review` so that they're easier to spot on the one global list of feature flags, but that's not something we need to worry about here :)

Reply:
Yep, agree, was going to move that config path down into ReportRecipientsReader, will do

Reply:
I implemented drews exact suggestion, moved the path of the rx ymls down into the reader itself